src/anniversary/sf33rd/Source/Game/IOConv.c

```diff
@@ -0,0 +1,106 @@
+#include "unknown.h"
+#include "common.h"
+#include "sf33rd/AcrSDK/common/mlPAD.h"
+
+extern u32 ioconv_table[24][2];
+
+void keyConvert(void) {
+    typedef struct {
+        u8 state;
+        u8 anstate;
+        u16 kind;
+        u32 sw;
+        u32 sw_old;
+        u32 sw_new;
+        u32 sw_off;
+        u32 sw_chg;
+        u32 sw_repeat;
+        PAD_STICK stick[2];
+    } IOPad;
+    IOPad* pad;    
+    u32 currSw;
+    s32 i;
+    s32 j;
+    s32 repeat_on;
+
+    repeat_on = 0;
+    if ((Debug_w[0x2B]) && (mpp_w.inGame) && (Game_pause == 0)) {
+        repeat_on = 1;
+    }
+    if ((save_w[Present_Mode].extra_option.contents[0][4]) && (mpp_w.inGame) && (Game_pause == 0)) {
+        repeat_on = 1;
+        if (((task[3].condition) == 1) && ((task[3].r_no[0]) != 0xA)) {
+            repeat_on = 0;
+        }
+    }
+
+    for (i = 0; i < 2; i++) {
+        flPADSetRepeatSw(&(*flpad_adr)[i], 0xFF000FU, 0xFU, 3U);

```

Comment:
Redundant unsigned literals. Normal int literals will suffice
```suggestion
        flPADSetRepeatSw(&(*flpad_adr)[i], 0xFF000F, 0xF, 3);
```

Reply:
Also `flpad_adr` is pretty much a 2D-array and should probably be treated as one:
```suggestion
        flPADSetRepeatSw(&flpad_adr[0][i], 0xFF000FU, 0xFU, 3U);
```

The same goes for every other mention of `flpad_adr` in this file